src/bindings/python/src/pyopenvino/graph/passes/manager.cpp

```diff
@@ -69,10 +68,9 @@ void regclass_passes_Manager(py::module m) {
     manager.def(
         "register_pass",
         [](ov::pass::Manager& self, const std::string& pass_name) -> void {
-            PyErr_WarnEx(PyExc_DeprecationWarning,
-                         "register_pass with this arguments is deprecated! "
-                         "Please use register_pass(ConstantFolding()) instead.",
-                         1);
+            Common::utils::deprecation_warning("register_pass(pass_name)",
+                                               "",
+                                               "Please use register_pass(ConstantFolding()) instead.");
```

Comment:
Can this be tested as well?

Reply:
I cannot see any c++ tests (except mocks) in the python bindings directory. I think this function is indirectly tested by testing python binding and decorator. However, if you want a separate test, please point me to the right place for such a test.